LICENSE-binary

```diff
@@ -271,6 +271,7 @@ org.eclipse.jetty:jetty-server
 org.eclipse.jetty:jetty-servlet
 org.eclipse.jetty:jetty-util-ajax
 org.eclipse.jetty:jetty-util
+org.eclipse.jetty:jetty-proxy
```

Comment:
The LICENSE-binary should reflect the content of binary tarball, so `jetty-client` should be listed as well